packages/zowe-explorer/README.md

```diff
@@ -23,26 +23,24 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.11.0](#whats-new-in-zowe-explorer-1.11.0)
+- [What's new in Zowe Explorer 1.12.1](#whats-new-in-zowe-explorer-1.12.1)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.11.0
-
-New features:
-
-- Added login and logout functions for base profiles. You can now log in to API Mediation Layer and generate a token for your base profile.
+## What's new in Zowe Explorer 1.12.1
```

Comment:
I thought we're only doing release notes for major versions and not for hot fixes?

Reply:
Maybe we can keep the 1.12.0 notes then add the bug fixes for 1.12.1. These 1.12.1 fixes should also be noted in the changelog

Reply:
My thinking was that if a patch version includes several fixes, we should include this information in the what's new section. @jellypuno @zdmullen  What are your thoughts? 

Good point about the changelog. Let me update it, too.